cpp/benchmarks/stream_compaction/distinct.cpp

```diff
@@ -65,6 +66,7 @@ using data_type = nvbench::type_list<int32_t, int64_t>;
 NVBENCH_BENCH_TYPES(nvbench_distinct, NVBENCH_TYPE_AXES(data_type))
   .set_name("distinct")
   .set_type_axes_names({"Type"})
+  .add_float64_axis("null_probability", {0.01})
```

Comment:
Should we add more values to this axis (for baseline signals) or this is just to modify it at runtime for particular cases?